recipe/meta.yaml

```
@@ -40,18 +40,17 @@ requirements:
     - python
     - pip
     - setuptools
-    - cython >=0.24.0
-    - fastrlock >=0.3
+    - cython >=0.29.22
```

Nitpick: Building CuPy will fail with Cython 3, so let's limit it for the time being (cupy/cupy#4610).
```suggestion
    - cython >=0.29.22,<3
```